tests/Unit/Domain/Creators/Test_Sphere.cpp

```diff
@@ -59,324 +63,321 @@ create_boundary_condition() {
-                                &inner_cube_sphericity,
-                                &use_equiangular_map](const auto frame) {
-    using TargetFrame = std::decay_t<decltype(frame)>;
-    auto local_coord_maps = make_vector_coordinate_map_base<Frame::BlockLogical,
-                                                            TargetFrame>(
-        Wedge3DMap{inner_radius, outer_radius, inner_cube_sphericity, 1.0,
-                   OrientationMap<3>{}, use_equiangular_map},
-        Wedge3DMap{inner_radius, outer_radius, inner_cube_sphericity, 1.0,
-                   OrientationMap<3>{std::array<Direction<3>, 3>{
-                       {Direction<3>::upper_xi(), Direction<3>::lower_eta(),
-                        Direction<3>::lower_zeta()}}},
-                   use_equiangular_map},
-        Wedge3DMap{inner_radius, outer_radius, inner_cube_sphericity, 1.0,
-                   OrientationMap<3>{std::array<Direction<3>, 3>{
-                       {Direction<3>::upper_xi(), Direction<3>::upper_zeta(),
-                        Direction<3>::lower_eta()}}},
-                   use_equiangular_map},
-        Wedge3DMap{inner_radius, outer_radius, inner_cube_sphericity, 1.0,
-                   OrientationMap<3>{std::array<Direction<3>, 3>{
-                       {Direction<3>::upper_xi(), Direction<3>::lower_zeta(),
-                        Direction<3>::upper_eta()}}},
-                   use_equiangular_map},
-        Wedge3DMap{inner_radius, outer_radius, inner_cube_sphericity, 1.0,
-                   OrientationMap<3>{std::array<Direction<3>, 3>{
-                       {Direction<3>::upper_zeta(), Direction<3>::upper_xi(),
-                        Direction<3>::upper_eta()}}},
-                   use_equiangular_map},
-        Wedge3DMap{inner_radius, outer_radius, inner_cube_sphericity, 1.0,
-                   OrientationMap<3>{std::array<Direction<3>, 3>{
-                       {Direction<3>::lower_zeta(), Direction<3>::lower_xi(),
-                        Direction<3>::upper_eta()}}},
-                   use_equiangular_map});
-    if (inner_cube_sphericity == 0.0) {
-      if (use_equiangular_map) {
-        local_coord_maps.emplace_back(
-            make_coordinate_map_base<Frame::BlockLogical, TargetFrame>(
-                Equiangular3D{
-                    Equiangular(-1.0, 1.0, -1.0 * inner_radius / sqrt(3.0),
-                                inner_radius / sqrt(3.0)),
-                    Equiangular(-1.0, 1.0, -1.0 * inner_radius / sqrt(3.0),
-                                inner_radius / sqrt(3.0)),
-                    Equiangular(-1.0, 1.0, -1.0 * inner_radius / sqrt(3.0),
-                                inner_radius / sqrt(3.0))}));
+  const auto& blocks = domain.blocks();
+  const auto block_names = sphere.block_names();
+  const size_t num_blocks = block_names.size();
+  const auto all_boundary_conditions = sphere.external_boundary_conditions();
+  double current_time = 1.0;
```

Comment:
Take this as an argument to the function. Then I think you can delete the time dep, expir time, and FoT arguments by just assuming a uniform translation and defaulting velocity to zero.